add beneficial reuse document (#259)
project-pareto/project-pareto: docs/model_library/beneficial_reuse/index.rst

.. _beneficial_reuse_model_in_pareto:

Beneficial Reuse Model in PARETO
================================
In the PARETO tool, beneficial reuse is represented as a unique kind of demand node (:math:`o \in O`). These nodes receive streams from various other nodes in the network, which include treatment or process sites, production sites, or storage sites. The distinct characteristic of these beneficial reuse nodes, which sets them apart from the rest of the produced water network, is their ability to accommodate various materials based on specific reuse requirements.

Rather than handling only produced water or clean brine of various qualities, beneficial reuse nodes can accept a diverse array of streams. These might include untreated produced water, clean brine, desalinated water, and concentrated brine, as well as recovered resources in different material states—gas, liquid, or solid. 

.. math::

    \sum_{l \in L | (l, o) \in LOA \cup LOT}F_{l,o,t} = F_{o,t}

.. math::

    \sum_{l \in L | (l, o) \in LOA \cup LOT} F_{l,o,t} \cdot Q_{l,qc,t} = Q_{o,qc,t} \cdot F_{o,t}

.. math::

   LOA = \\{(l, o) \in LLA \ | \ o \in O\\}

.. math::

>    LRT = \\{(l, o) \in LLT \ | \ o \in O\\}

Reviewer: ```suggestion
   LOT = \\{(l, o) \in LLT \ | \ o \in O\\}
```
Is this supposed to be LOT?

Water quality requirements for DAC systems resemble those for industrial cooling. However, considering the availability and quality of produced water sources is crucial for the viability and environmental benefits of DAC technology.

Hydrogen production
-------------------------------
The potential for beneficial reuse of produced water in hydrogen hubs holds great promise for addressing water scarcity and supporting the energy transition. Water consumption in hydrogen production varies depending on the type of hydrogen and cooling requirements. Green hydrogen production typically requires around 9 kg of water per kilogram of hydrogen, but when considering cooling demands, the average water consumption increases to approximately 18.08 kg per kilogram of hydrogen. Blue hydrogen production has a stoichiometric ratio of about 5.5 kg of water per kilogram of hydrogen, which rises to approximately 21.8 kg when factoring in cooling requirements.

> In the context of green hydrogen production, the use of ultra-pure water is crucial to ensure efficient operation. This involves employing deionized and demineralized water to minimize the presence of cations, such as Fe3+, Mg2+, Ca2+, Cu2+, and Na+, that can have detrimental effects on the hydrogen production process. The presence of these divalent ions can negatively impact the performance and durability of key components, such as proton exchange membranes (PEM), in hydrogen electrolyzers. Similarly, blue hydrogen production requires pure water with low total dissolved solids (TDS) for optimal performance.

Reviewer 2: ```suggestion
In the context of green hydrogen production, the use of ultra-pure water is crucial to ensure efficient operation. This involves employing deionized and demineralized water to minimize the presence of cations, such as Fe\ :sup:`3+`, Mg\ :sup:`2+`, Ca\ :sup:`2+`, Cu\ :sup:`2+`, and Na\ :sup:`+`, that can have detrimental effects on the hydrogen production process. The presence of these divalent ions can negatively impact the performance and durability of key components, such as proton exchange membranes (PEM), in hydrogen electrolyzers. Similarly, blue hydrogen production requires pure water with low total dissolved solids (TDS) for optimal performance.
```

Considering the DOE's target of producing green hydrogen for less than $1 per kilogram, it is important to ensure that the treatment and transportation of produced water, does not hinder this objective. Integrating produced water from oil and gas operations into hydrogen hubs offers a viable solution, particularly in highly water-stressed regions. This approach repurposes water that would otherwise be disposed of, reducing the pressure on freshwater resources and minimizing the environmental impacts associated with sea water desalination plants.

> The hydrogen explorer map, accessible via the provided link, provides valuable insights into the distribution and capacity requirements of proposed hydrogen hubs across different regions.

Reviewer 2: Where is the link that is referenced?

Solution Mining
---------------------------

Solution mining employs the technique of dissolving water-soluble minerals found beneath the Earth's surface, such as potash, salt, and trona, using water circulated through drilled wells. An innovative strategy within this field utilizes heated brine rather than fresh water for preferential solution mining, specifically useful in selectively retrieving potassium chloride from sylvinite deposits. This process, however, is water-intensive, and can consume millions of cubic meters of water annually depending on the scale and size of the mining operation. For instance, potash mining could require 10-15 million cubic meters of water annually. Produced water could potentially serve as an alternative water source for this operation. However, its use must be carefully managed to prevent mine contamination, necessitating further research and careful regulation for sustainable implementation.

Resource  recovery
------------------------------------

> Produced water (PW) is increasingly being recognized as a potential source of valuable resources and critical minerals. Studies have highlighted the potential economic feasibility of extracting elements such as lithium, magnesium, strontium, and ammonium from PW, with potential yearly revenues per well reaching up to $250,000 per well in the Permian basin. 

Reviewer 2: ```suggestion
Produced water is increasingly being recognized as a potential source of valuable resources and critical minerals. Studies have highlighted the potential economic feasibility of extracting resources such as lithium, magnesium, strontium, and ammonium from PW, with potential yearly revenues per well reaching up to $250,000 per well in the Permian basin. 
```
1) Introduce the abbreviation "PW" the first time that "produced water" is mentioned in the page.

2) I'm nit picking here, but ammonium is not an element. Also, do you mean to say ammonia instead of ammonium?

Emerging technologies such as adsorption, ion exchange, membrane-based technologies, and electrodialysis are playing a crucial role in resource recovery. These technologies, when effectively implemented, can enhance the efficiency and selectivity of resource recovery, turning PW from a waste stream into a multi-faceted resource. 

> For instance, a recent study demonstrated the feasibility of simultaneously recovering NH4+, K+ and Mg2+ from PW by struvite precipitation after calcium pretreatment. The process resulted in NH4+, K+ and Mg2+ recovery efficiencies of 85.9%, 24.8% and 96.8%, respectively, providing an example of a recovery-centered approach.

Reviewer 2: ```suggestion
For instance, a recent study demonstrated the feasibility of simultaneously recovering NH\ :sub:`4`\ :sup:`+`, K\ :sup:`+` and Mg\ :sup:`2+` from PW by struvite precipitation after calcium pretreatment. The process resulted in NH\ :sub:`4`\ :sup:`+`, K\ :sup:`+` and Mg\ :sup:`2+` recovery efficiencies of 85.9%, 24.8% and 96.8%, respectively, providing an example of a recovery-centered approach.
```

> However, the large-scale implementation of these technologies is dependent on the net value of recovered resources, which is a function of the market price of recovered elements, the concentration in the brine, and the extraction cost of any particular element. Therefore, the market consumption/demand of certain elements available from the waste brine must also be considered while determining the economic potential of resource recovery.

Reviewer 2: ```suggestion
However, the large-scale implementation of these technologies is dependent on the net value of recovered resources, which is a function of the market price of recovered resources, the concentration in the brine, and the extraction cost of any particular resource. Therefore, the market consumption/demand of certain resources available from the waste brine must also be considered while determining the economic potential of resource recovery.
```
